engine/src/main/java/org/terasology/math/Rotation.java

```diff
@@ -79,19 +82,40 @@ public static Rotation rotate(Yaw yaw, Pitch pitch, Roll roll) {
         return ROTATIONS.get(indexFor(yaw, pitch, roll));
     }
 
+    public static Rotation findReverse(Rotation rotation) {
+        Side frontResult = rotation.rotate(Side.FRONT);
```

Comment:
I would suggest the reverse of a rotation should be the rotation that undoes the rotation. I'm not sure this code achieves this? If it does I think there is a likely a more direct way of achieving it.

I also assume that null shouldn't be a legitimate result, I would suggest a RuntimeException thrown instead.